docs/internals/guide.rst

```diff
@@ -186,6 +186,7 @@ Here's an example using Celery in single-mode:
 
 .. code-block:: python
 
+    # XXX: No longer valid?
```

Comment:
Yes this is no longer valid.

Reply:
That's what I figured. Is the "here's the same thing..." version still valid? Should I just rip out the first version and make that second sample the one-and-only?

Reply:
(I'll proceed on the assumption that both answers are "yes", and add a commit as described. It's easy enough to drop or tweak further if need be.)

Reply:
Hrm. Actually, reading the intro to this section, the whole point is to illustrate the difference between (as the heading _says_, point-blank): "Applications vs. 'single mode'".So it needs two examples, if illustrating that dichotomy is still desirable at all. If not, I can just pull the entire section out.

If it _is_ worth illustrating, it'll need a new first (single-mode) example. Can the current Tasks API still be used single-mode style; is it just a matter of updating the imports?

Reply:
@thedrow / @auvipy Any thoughts on this? I believe it's the last outstanding issue. For context, this is the documentation section that reads:

> #### Applications vs. “single mode”
> In the beginning Celery was developed for Django, simply because this enabled us get the project started quickly, while also having a large potential user base.
> 
> In Django there’s a global settings object, so multiple Django projects can’t co-exist in the same process space, this later posed a problem for using Celery with frameworks that don’t have this limitation.
> 
> Therefore the app concept was introduced. When using apps you use ‘celery’ objects instead of importing things from Celery sub-modules, this (unfortunately) also means that Celery essentially has two API’s.
> 
> Here’s an example using Celery in single-mode:
> 
> ```
> from celery import task
> from celery.task.control import inspect
> 
> from .models import CeleryStats
> 
> @task
> def write_stats_to_db():
>     stats = inspect().stats(timeout=1)
>     for node_name, reply in stats:
>         CeleryStats.objects.update_stat(node_name, stats)
> 
> ```
> and here’s the same using Celery app objects:
> 
> ```
> from .celery import celery
> from .models import CeleryStats
> 
> @app.task
> def write_stats_to_db():
>     stats = celery.control.inspect().stats(timeout=1)
>     for node_name, reply in stats:
>         CeleryStats.objects.update_stat(node_name, stats)
> ```
> 
> In the example above the actual application instance is imported from a module in the project, this module could look something like this:
> 
> ```
> from celery import Celery
> 
> app = Celery(broker='amqp://')
> 
> ```

The issue being that the first example is no longer valid code, since the removal of `celery.task`. I am not qualified to write a replacement snippet (or even to determine whether it's still necessary to have one), but I'll be happy to paste one in to the docs if someone can supply it.

Reply:
I removed the added comment I'd placed here, as well. Either the first example will get updated, or it won't; since it's only a piece of example code in the documentation, it doesn't affect anything _operationally_ regardless, so this conversation can serve as indication that it needs to be addressed. (Unless someone feels like migrating it to a new Issue.)